UKHomeOffice/ide-controller
# COP 1976: Data read table context

## Description
This branch adds event context to the Data Read table, indicating to the user the success state of a document scan. 

## To test
- pull repo
- run `npm ci && npm run dev`
- If the electron window displays blank, refresh with `Cmd + r` or `Ctrl + r`
- To mock a document scan, press `F1`
- The "Open Chip" and "Read MRZ" columns should display the "Successful" tag. 

## Developer Checklist
\* Required
